File: Decreto.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main(){
    int ingreso;
    double tributo;
    cout << "Tienes que aportar al tesoro real, ¿cuanto ganas?" << endl;
    cin >> ingreso;
    
    if (ingreso < 0){
        cout << "¿Cómo vas a ganar monedas negativas?" << endl;
        tributo = 0;
    }else if (ingreso <= 10000){
        tributo = 0.0;
    }else if (ingreso <= 50000){
        tributo = (ingreso - 10000) * 0.1;
    }else{
        tributo = 4000 + ((ingreso - 50000) * 0.2);
    }
    cout << "Tienes que pagar " << fixed << setprecision(2) << tributo << " monedas al tesoro real.";
    
    return 0;
}

File: Decreto.py
print("Ingrese las monedas de oro ganadas en un año: ")
ingreso = int(input())

if ingreso < 0:
    tributo = "¿Cómo vas a ganar monedas negativas?"
elif ingreso <= 10000:
    tributo = "No tienes que pagar nada"
elif ingreso <= 50000:
    tributo = f"Tienes que pagar {(ingreso - 10000) * 0.1} monedas al tesoro real "
else:
    tributo = f"Tienes que pagar {4000 + ((ingreso - 50000) * 0.2)} al tesoro real"
    
print(tributo)